### PR TITLE
disable more sentry events

### PR DESCRIPTION
**Pull request by snopoke**
Stop reporting these events:

https://sentry.io/organizations/dimagi/issues/1216197130/
https://sentry.io/organizations/dimagi/issues/1216241634/

### CODE REVIEW

**Comment by snopoke** [comment]
@mkangia bump

File: settingshelper.py [hunk 1]
```diff
@@ -251,6 +251,7 @@ def _before_send(event, hint):
     ignore_logger('django.template')
     ignore_logger('pillowtop')
     ignore_logger('restore')
+    ignore_logger('kafka.conn')
```
Comment (mkangia) [inline settingshelper.py]: is this logger actually in use?

Comment (snopoke) [reply settingshelper.py]: https://sentry.io/organizations/dimagi/issues/1216197130/

Comment (mkangia) [reply settingshelper.py]: okay. i was searching for `kafka.conn` in HQ codebase but thats probably not correct. Is there a way to search in code base to see where it's being used?

Comment (snopoke) [reply settingshelper.py]: It is used the in the kafka library so you would have to search your installed python files as well

Comment (mkangia) [reply settingshelper.py]: thanks.

**Pull request merged**